portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -48,8 +50,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   /** Accepts a POST request containing a new marker. */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    double lat = Double.parseDouble(request.getParameter("lat"));
-    double lng = Double.parseDouble(request.getParameter("lng"));
+    double lat = Double.parseDouble(request.getParameter(ENTITY_PROPERTY_KEY_1));
```

Comment:
This change (and the next line) is wrong. `ENTITY_PROPERTY_KEY_1` may be equal to 'lat' now, but it's meant to be the key for storing the data into the data service. The idea is that if we change the key for storing data in the api later, we can change one constant and the code would continue to work because both `entity.getProper(ENTITY_PROPERTY_KEY_1, ...)` and `entity.setProperty(ENTITY_PROPERTY_KEY_1, ...)` use it.

This code is parsing the key from the request, which is logically different. If we change the key in future, this would break because the request from the js hasn't changed.